adapters/zkdx/src/index.ts

```diff
@@ -0,0 +1,106 @@
+import fs from 'fs';
+import {write} from 'fast-csv';
+import {gql, GraphQLClient} from "graphql-request";
+import {BigNumber, ethers} from "ethers";
+
+const graphClient = new GraphQLClient("https://api.studio.thegraph.com/query/47302/zkdx-graph-linea/v0.0.6")
+const lineaProvider = new ethers.providers.JsonRpcProvider("https://rpc.linea.build");
+
+const getBalanceShots = gql`
+    query getBalanceShots($block_number: Int) {
+        tokenBalanceShots(
+            first: 1000
+            orderBy: block_number
+            orderDirection: desc
+            where: {
+                block_number_lte: $block_number
+            })
+        {
+            user_address
+            token_address
+            token_balance
+            block_number
+        }
+    }`
+
+export const getUserTVLByBlock = async (block_number: number) => {
```

Comment:
@helgaxdev please check the readme and make sure the input, output and Function name matches the expectation,

Reply:
fixed